infra-templates/nfs/3/rancher-compose.yml

```diff
@@ -1,23 +1,23 @@
 .catalog:
   name: "Rancher NFS"
-  version: "0.2.2"
+  version: "0.3.0"
   description: |
     Docker volume plugin for NFS
-  minimum_rancher_version: v1.4.0-rc1
+  minimum_rancher_version: v1.5.0-rc1
```

Comment:
let's make this v1.6.0 for now. and then can you add rancher max version on the v0.2.2 to set for v1.5.99?